stl/inc/spanstream

```diff
@@ -218,7 +218,8 @@ public:
             !convertible_to<_ReadOnlyRange, _STD span<_Elem>> && convertible_to<_ReadOnlyRange, _STD span<const _Elem>>)
     explicit basic_ispanstream(_ReadOnlyRange&& _Range)
         : basic_ispanstream(
-            _STD span<_Elem>{const_cast<_Elem*>(_RANGES data(_Range)), static_cast<size_t>(_RANGES size(_Range))}) {}
+            _STD span<_Elem>{const_cast<_Elem*>(static_cast<_STD span<const _Elem>>(_Range).data()),
+                static_cast<size_t>(static_cast<_STD span<const _Elem>>(_Range).size())}) {}

```

Comment:
The return type `span<T, Ext>::size` is already `size_t`, so there shouldn't be `static_cast`.

Reply:
I have fixed it.